lib/utils/http.go

```diff
@@ -176,3 +178,52 @@ func (m *maxBytesReader) Read(p []byte) (int, error) {
 	}
 	return n, err
 }
+
+// Sensitive headers list based on team discussion and Akami's list of
+// not-to-be-logged headers here:
+//    https://techdocs.akamai.com/edge-diagnostics/reference/sensitive-headers
+//
+//  - Authorization
+//  - Proxy-Authorization
+//  - Set-Cookie
+//  - Anything containing "API-Key"
+//  - X-Amz-Security-Token
+
+// sensitiveHeaderKeys is the list HTTP headers deemed to be too sensitive
+// to be written to a log.
+var sensitiveHeaderKeys []string = []string{
```

Comment:
```suggestion
var sensitiveHeaderKeys = []string{
```

No need to repeat the type twice so close together.